src/commands/moderation/warn/deletewarn.ts

```diff
@@ -0,0 +1,64 @@
+import { CommandDefinition } from '../../../lib/command';
+import { Roles, CommandCategory } from '../../../constants';
+import { makeEmbed } from '../../../lib/embed';
+import { getConn } from '../../../lib/db';
+
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MODERATION_TEAM,
+];
+
+const noPermEmbed = makeEmbed({
+    title: 'Warn',
+    description: 'You do not have permission to use this command.',
+    color: 'RED',
+});
+
+const deleteEmbed = makeEmbed({
+    title: 'Warn - Removed',
+    description: 'Warning has been removed',
+    color: 'GREEN',
+});
+
+const removeFailed = makeEmbed({
+    title: 'Warn - Failed',
+    description: 'Warning could not be removed',
+    color: 'RED',
+});
+
+const noWarning = makeEmbed({
+    title: 'Warn - No Warning',
+    description: 'Could not find warning. Please check the `Warn ID`',
+    color: 'RED',
+});
+
+export const deletewarn: CommandDefinition = {
```

Comment:
Camelcase filename and const

Reply:
We have a mixture of camelcase and non - will have to refactor the rest